wiki/History_of_osu!/osu!_wiki/en.md

```diff
@@ -1,36 +1,36 @@
----
-outdated: true
----
```

Comment:
maybe add `needs_cleanup` though cuz a lot of these entries seems kinda sloppily written and some exact dates are missing for whatever reason